tests/integration/sync/test_sync_build_in_source.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pytest
+import logging
+import json
+from unittest import skip
+from parameterized import parameterized
+
+from tests.integration.sync.sync_integ_base import SyncIntegBase
+from tests.testing_utils import run_command_with_input
+from samcli.lib.utils.resources import AWS_LAMBDA_FUNCTION
+
+LOG = logging.getLogger(__name__)
+
+
+@skip("Building in source option is not exposed yet. Stop skipping once it is.")
```

Comment:
skipping since the click option to build in source is not exposed yet. Made sure tests passed on my machine when I exposed the option and didn't skip the tests.